sympy/physics/units/tests/test_quantities.py

```diff
@@ -528,6 +528,20 @@ def test_issue_22819():
     assert Da.scale_factor == 1.66053906660000e-24
 
 
+def test_issue_20288():
+    from sympy import sympify
+    from sympy.core.numbers import E
+    from sympy.physics.units import energy
+    u = Quantity('u')
+    v = Quantity('v')
+    SI.set_quantity_dimension(u, energy)
+    SI.set_quantity_dimension(v, energy)
+    u.set_global_relative_scale_factor(1, joule)
+    v.set_global_relative_scale_factor(1, joule)
+    expr = 1 + exp(u**2/v**2)
+    assert SI._collect_factor_and_dimension(expr) == (sympify(1 + E), Dimension(1))
```

Comment:
why do you need to sympify this expression?

Reply:
It was actually unnecessary, I tried to mimic a test . Thanks for pointing it out ! It has been removed in the latest commit .